### PR TITLE
Drbg spec updates

### PR DESCRIPTION
**Pull request by celic**
Updates the registration to use more compound objects allowing for multiple 'modes' to be submitted at the same time.

### CODE REVIEW

File: src/acvp_sub_drbg.xml [hunk 1]
```diff
@@ -741,88 +756,294 @@ the bit lengths the supported values shall be specified explicitly. </t>
...
-                                 "max" : 512,
-                                 "step" : 256} ],
-              "additionalInputLen": [ { "min": 0,
-                                 "max" : 512,
-                                 "step" : 256} ],
-                "returnedBitsLen": 1024
-            }
+{
+  "Algorithm": "ctrDRBG",
+  "Mode": "",
```
Comment (Kritner) [inline src/acvp_sub_drbg.xml]: Remove empty property?

File: src/acvp_sub_drbg.xml [hunk 2]
```diff
@@ -741,88 +756,294 @@ the bit lengths the supported values shall be specified explicitly. </t>
...
-                "reseedImplemented" : true,
-                "entropyInputLen": [ 256 ],
-                "nonceLen": [ 128 ],
-                "persoStringLen": [ 256 ],
-               "additionalInputLen": [256 ],
-                "returnedBitsLen": 1024
-            }
+        {
+  "Algorithm": "hashDRBG",
+  "Mode": "",
```
Comment (Kritner) [inline src/acvp_sub_drbg.xml]: Empty property

**Comment by atvassilev** [comment]
One more thing: the examples list "acvVersion" :  "0.3". This should be corrected to 0.4. Please update the date of the document as well. 

I realize this tedious work but for now we don't have a better mechanism to keep track of these details.

**Comment by bfussell** [comment]
Seems the new examples don't follow the expected camel case used in the definitions.
There are also keyword mismatches, such as prereqVals and the example has PrereqValues.

**Pull request merged**